village_add_thanks.php

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+
+<meta property="fb:appid" content="<?php print FACEBOOK_APP_ID; ?>"/>
+<meta property="og:image" content="<?php print PICTURES_DIR.$bannerPicture; ?>"/>  <!-- here could we put the pic uploaded by the user? -->
+<meta property="og:title" content="I just added <?php print $villageName; ?> to Village X's map!"/>
+<meta property="og:url" content="<?php print BASE_URL.$projectId; ?>"/>  <!-- not sure what to put here -->
+<meta property="og:description" content="Village X maps villages and funds development projects chosen by them."/>
+
+<?php 
+$metaProvided = 1; 
+include('header.inc'); ?>
+
+<div class="row" style="height:90vh;">
+
+<div class="col s12 m12 l6" style="height:100%;padding:7% 6% 6% 6%">
+	<div class="valign-wrapper">
+		<div>
+        		<div class="row left-align">
+        			<h3 class="header col s12 black-text text-lighten-2 text-shadow: 2px 2px 7px #111111 flow-text" style="font-weight: 300">Thanks, (name), for adding your village in (country) on (date)!</h3>
```

Comment:
Don't put style tags in the class list - please remove all of these.

Reply:
i think that those tags are class tags.

Reply:
except for text-shadow